gateway/handle_observability.go

```diff
@@ -18,14 +20,13 @@ func (gw *Handle) NewSourceStat(arctx *gwtypes.AuthRequestContext, reqType strin
 	}
 }
 
-func (gw *Handle) newSourceStatTagsWithReason(arctx *gwtypes.AuthRequestContext, reqType, reason string) stats.Tags {
+func (gw *Handle) newSourceStatTagsWithReason(s backendconfig.SourceT, reqType, reason string) stats.Tags {
```

Comment:
Didn't get why we need to pass too much information when `*gwtypes.AuthRequestContext` was sufficient?

Reply:
Why we are not passing `source_type` in tags?

Reply:
missed it added now